nfpm.yaml

```diff
@@ -229,6 +229,18 @@ contents:
   - src: rpi-package-download/rpi-package-download
     dst: /usr/local/bin/rpi-package-download
 
+  - src: monitor/monitor.sh
+    dst: /usr/local/bin/monitor.sh
+
+  - src: monitor/monitor.py
+    dst: /usr/share/rpi-sb-provisioner/monitor.py
+
+  - src: monitor/systemctl_python.py
+    dst: /usr/share/rpi-sb-provisioner/systemctl_python.py
```

Comment:
Please put these python files into a subdirectory, so they're logically grouped. We'll have other apps installed too.